lib/Factory/ModuleXmlTrait.php

```diff
@@ -280,6 +281,27 @@ private function parseElements(\DOMNodeList $elementsNodes): array
         return $elements;
     }
 
+    /**
+     * @param \DOMNodeList $legacyTypeNodes
+     * @return \Xibo\Widget\Definition\Property[]
+     */
+    private function parseLegacyTypes(\DOMNodeList $legacyTypeNodes): array
+    {
+        $legacyTypes = [];
+        foreach ($legacyTypeNodes as $node) {
+            /** @var \DOMNode $node */
+            if ($node instanceof \DOMElement) {
+                $legacyType = new LegacyType();
+                $legacyType->name = $node->textContent;
```

Comment:
We should `trim()` this in case there are some stray characters in the nodes text content.